dubbo-metadata/dubbo-metadata-report-redis/pom.xml

```diff
@@ -41,9 +41,8 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.github.kstyrc</groupId>
+            <groupId>com.github.codemonstur</groupId>
             <artifactId>embedded-redis</artifactId>
-            <scope>test</scope>
```

Comment:
why remove test scope

Reply:
because it is already set at https://github.com/martin-g/dubbo/blob/fix-build-and-tests-on-arm64/dubbo-dependencies-bom/pom.xml#L628 in the dependencyManagement section

Reply:
this way it is in sync with 
- https://github.com/martin-g/dubbo/blob/fix-build-and-tests-on-arm64/dubbo-registry/dubbo-registry-multiple/pom.xml#L55-L58
- https://github.com/martin-g/dubbo/blob/fix-build-and-tests-on-arm64/dubbo-registry/dubbo-registry-redis/pom.xml#L47-L50
- https://github.com/martin-g/dubbo/blob/fix-build-and-tests-on-arm64/dubbo-rpc/dubbo-rpc-redis/pom.xml#L42-L45

Reply:
Rather than remove `<scope>test</scope>`, add `<scope>test</scope>` will be better I think.
* https://github.com/martin-g/dubbo/blob/fix-build-and-tests-on-arm64/dubbo-registry/dubbo-registry-multiple/pom.xml#L55-L58
* https://github.com/martin-g/dubbo/blob/fix-build-and-tests-on-arm64/dubbo-registry/dubbo-registry-redis/pom.xml#L47-L50
* https://github.com/martin-g/dubbo/blob/fix-build-and-tests-on-arm64/dubbo-rpc/dubbo-rpc-redis/pom.xml#L42-L45

Reply:
@horizonzy OK, I will add the scope!

Reply:
Done!
Also rebased to latest `master`